examples/mnist/train.cpp

```diff
@@ -42,6 +46,7 @@ static void construct_net(network<sequential>& nn) {
         X, X, O, O, O, X, X, O, O, O, O, X, O, O, X, O,
         X, X, X, O, O, O, X, X, O, O, O, O, X, O, O, O
     };
+// clang-format on
```

Comment:
The indentations of `// clang-format off` comment and `// clang-format on` comment are different.
Maybe adding 4 or 2 spaces before `// clang-format on` comment would help.